velox/expression/EvalCtx.cpp

```diff
@@ -49,16 +49,15 @@ EvalCtx::EvalCtx(core::ExecCtx* execCtx)
   VELOX_CHECK_NOT_NULL(execCtx);
 }
 
-void EvalCtx::setWrapped(
+VectorPtr EvalCtx::applyWrapToPeeledResult(
     Expr* FOLLY_NONNULL expr,
```

Comment:
nit: It looks like expr is only used through expr->type(). Could this function just receive a TypePtr instead of Expr*? (I can change it later if it's not easy to change in this PR.)

Reply:
done